semantic_conventions/trace/messaging.yaml

```
@@ -260,3 +260,66 @@ groups:
               brief: 'Broadcasting consumption model'
         brief: >
           Model of message consumption. This only applies to consumer spans.
+
+  - id: messaging.pulsar
+    prefix: messaging.pulsar
+    extends: messaging
+    brief: >
+      Attributes for Apache Pulsar
+    attributes:
+      - id: subscription
+        type: string
+        brief: >
+          Subscription name of Pulsar consumer.
```

I read more on https://pulsar.apache.org/docs/next/concepts-messaging#subscriptions and I'm not sure this `brief` is accurate. The docs says:

> A subscription is a named configuration rule that determines how messages are delivered to consumers

So I think we change this to be:

```suggestion
          The Pulsar subscription name
```